index.html

```diff
@@ -276,45 +327,65 @@ <h4 class="stack-title">Skills</h4>
       </section>
 
       <!-- contact section -->
-      <div id="contact-section">
-        <div id="background-img">
-          <img src="./images/contact-background.svg" alt="contact-background">
-        </div>
-        <h3>
-          I'm always interested in hearing about new projects, so if you'd like
-          to chat please get in touch.
-        </h3>
-        <form
-          id="contact-form"
-          action="https://formspree.io/f/xoqrpjkd"
-          method="post"
-        >
-          <fieldset>
-            <input
-              type="text"
-              id="name"
-              placeholder="Full Name"
-              maxlength="30"
-              required
+      <section id="contact-section">
+        <div id="top">
+          <div id="background-img">
+            <img
+              src="./images/contact-background.svg"
+              alt="contact-background"
```

Comment:
- [ ] Please adjust your contact section to be centered just like the Figma design on screen sizes from `768px` and above 👇 
### Your Design
![image](https://user-images.githubusercontent.com/46839548/165747508-11033bed-ca7a-446c-b1b3-896d3e8a6d43.png)
### Figma Design
![image](https://user-images.githubusercontent.com/46839548/165747327-32448d2e-a583-43c2-93a1-f46853b608d1.png)